test/helper/helper.go

```diff
@@ -360,6 +361,68 @@ func CleanTestGateway(t *testing.T, gatewayId string) error {
 	if err != nil {
 		return fmt.Errorf("Error when calling `MetalGatewaysApi.DeleteMetalGateway`` for %v: %v\n", gatewayId, err)
 	}
+	return nil
+}
+
+func CreateTestInterConnection(projectId, name string) (string, error) {
```

Comment:
These new create and clean helpers should be updated to match all of the other helpers, which means they should: - take a `*testing.T` as an argument
- register themselves as test helpers
- call `t.Fatalf` if something breaks in order to fail the test and halt test execution; they should not return an error
- create helpers may return the object that was created instead of the ID
- create helpers should register the clean helpers by using `t.Cleanup`
- clean helpers should ignore 404 errors (it's fine if the object was already deleted)